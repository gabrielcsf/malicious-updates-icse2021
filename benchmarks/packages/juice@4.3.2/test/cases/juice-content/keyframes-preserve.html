<html>
<head>
    <style>
        p {
            color: blue;
            opacity: 0;
            animation: keyframeName 3s infinite;
        }
        @keyframes keyframeName {
            0%, 100% {
                opacity: 0;
            }
            50% {
                opacity: 1;
            }
        }
    </style>
</head>
<body>
    <p>hi</p>
</body>
</html>
