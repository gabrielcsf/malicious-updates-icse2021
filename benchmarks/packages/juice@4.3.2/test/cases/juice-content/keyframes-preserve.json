{
    "url": "./",
    "removeStyleTags": true,
    "preserveKeyFrames": true
}
