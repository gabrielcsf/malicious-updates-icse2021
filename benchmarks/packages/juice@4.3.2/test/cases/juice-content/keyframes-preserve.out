<html>
<head>
    <style>
@keyframes keyframeName {
  0%,
100% {
    opacity: 0;
  }

  50% {
    opacity: 1;
  }
}
</style>
</head>
<body>
    <p style="color: blue; opacity: 0; animation: keyframeName 3s infinite;">hi</p>
</body>
</html>
